Tasks/ExtractFilesV1/extractfilestask.ts

```diff
@@ -10,6 +10,9 @@ var cleanDestinationFolder: boolean = tl.getBoolInput('cleanDestinationFolder',
 var repoRoot: string = tl.getVariable('System.DefaultWorkingDirectory');
 tl.debug('repoRoot: ' + repoRoot);
 
+var overwriteDestinationVar: string = tl.getVariable('ExtractFile.OverwriteDestination');
+var overwriteDestination: boolean = overwriteDestinationVar ? overwriteDestinationVar.toLowerCase() === 'true' : true;
```

Comment:
Let's make this the opposite, let's default it to overwriting and only if the variable is set then we prevent it. This lets users opt-out if they want to but all other users get the overwriting.